### ISSUE TITLE
myservices.timewarnercable.com

### ISSUE THREAD

**Issue by the1wolf** [reek/anti-adblock-killer#2959]
https://www.timewarnercable.com/residential/order/session
Mozilla Firefox
Greasemonkey
Adblock Plus
EasyList + AakList

**Comment by uBlock-user** [comment]
Force update your filters, already patched by Easylist.

**Comment by the1wolf** [comment]
I did, still getting this.
![1](https://cloud.githubusercontent.com/assets/25043074/23972539/91fc2296-09a8-11e7-963c-d36d5251c628.gif)

**Comment by uBlock-user** [comment]
add - `.sitecatalyst.js` and `/sitecatalyst/*` and check if it happens again

**Comment by the1wolf** [comment]
I added those, but still getting it.

**Comment by uBlock-user** [comment]
I cannot reproduce it. Try `||$third-party,script,domain=timewarnercable.com`

**Comment by the1wolf** [comment]
nope, still there.  This happens when i log in and click 'upgrade services', then there is a box that says "please wait a moment, we are processing your request".  The above screen appears along with the yellow banner at the top of the screen.

**Comment by uBlock-user** [comment]
Remove cache and all site data for that site and try again.

**Comment by the1wolf** [comment]
That did it.  Thanks for all the assistance!
